documentation/testing_plans.md

```diff
@@ -0,0 +1,783 @@
+# Testing plans
+
+Bolt ships with a library of helpers, named `BoltSpec`, intended to be used for
+writing unit tests for plans with the `RSpec` testing tool. Before writing unit
+tests for plans, get familiar with [RSpec](https://rspec.info/).
+
+## Set up the test environment
```

Comment:
Hm, good question... I'm not sure what the best approach is since I don't think we should imply plans can only be tested in modules. Technically you can add simplified `Gemfile`, `Rakefile`, and `spec_helper.rb` files to your project manually, and then the tests will work normally. You should also be able to convert your project to a module to get those files from PDK, though you'd get a ton of extraneous files with them.

Maybe two subsections: "Testing plans in a module" and "Testing plans in a project"?

Reply:
Yeah, I think 2 sections makes sense